app/components/geolocation/tasks.rake

```diff
@@ -0,0 +1,6 @@
+desc "Import locations from a CSV file"
+task :import_locations, %i[filepath] => [:environment] do |_task, args|
```

Comment:
a rake-task (basically Ruby script) that can be run to import data. It can be used as a cron job. It will need to be adjusted to first get the CSV file from somewhere like a cloud storage e.g. S3.

The execution of this cron job can be configured in many ways:
1. Run it on the existing app server via `crontab`
2. Use a background job process (like `Sidekiq`) that support cron-like jobs
3. Use Kubernetes Automated Tasks if we use k8s
4. Use AWS Scheduled Tasks if we use ECS